src/Features/CSharp/Portable/ChangeSignature/CSharpChangeSignatureService.cs

```diff
@@ -156,7 +156,18 @@ public CSharpChangeSignatureService()
             }
 
             var symbolInfo = semanticModel.GetSymbolInfo(matchingNode, cancellationToken);
-            return (symbolInfo.Symbol ?? symbolInfo.CandidateSymbols.FirstOrDefault(), 0);
```

Comment:
previously the code gave up when it was on an invocation and set the selected parameter to 0.  In order to know which parameter to delete for the intent however we need to handle this case.

So we derive the parameter index from the selected argument.